versioned_docs/version-V2/guides/index.md

```diff
@@ -0,0 +1,11 @@
+### What is a relay?
+
+A relay is a third-party who receives a fee for including relayed transactions in the blockchain. (McMenamin, Daza, and Fitz. https://eprint.iacr.org/2022/155.pdf, p.3)
+
+Ethereum transactions are tied to a specific address, and transactions sent from
+this address consume Ether to pay for gas. When a signal is broadcast, the
+Ethereum address used for the transaction might be used to break the anonymity
+of the broadcaster. Semaphore allows signals to be received from any address,
+allowing a relayer to broadcast a signal on behalf of a user. Applications might
+provide rewards for relayers and implement front-running prevention mechanisms, such as requiring the signals to include the relayer’s address, binding the
+signal to that specific address.(https://semaphore.appliedzkp.org/whitepaper-v1.pdf, p.6)
```

Comment:
The paper refers to version 1, which in my opinion went beyond the protocol and tried to explain how the architecture of a dApp with Semaphore could be organized. The main problem is that the one who creates the zero-knowledge proof anonymously cannot be the same one who signs the Ethereum transaction, as the signer's address is public. The relayer may be an architectural solution.